js/notify.js

```diff
@@ -15,7 +15,7 @@ define([
             level: type == 'error' ? 'danger' : type,
             message: message
         }));
-        Log.debug('Notify:', {type: type, message: message, container: container});
+        log.debug('Notify:', {type: type, message: message, container: container});
```

Comment:
Nope, you're right. Need to investigate further why this causes a `Log.debug is not a function` error.